local_deploy/get_token.py

```diff
@@ -0,0 +1,15 @@
+import os
```

Comment:
This is for changing the environment variable in the main function. But we can move it in the __main__. Done in [ddc8ea1](https://github.com/UKP-SQuARE/square-core/pull/354/commits/ddc8ea14880c24fe5e5c22b02ec0822b29f4cfee)

Reply:
i meant this entire file, not the import 🙃
When the `SQUARE_PRIVATE_KEY_FILE` env var is not set, it will use the current dir and private_key.pem. See https://github.com/UKP-SQuARE/square-auth/blob/6410fe6d0318bf833ceefd288d4d69e350c2cb89/square_auth/utils.py#L11-L21